cloverage/test/resources/index.html

```diff
@@ -16,24 +16,24 @@
    </tr></thead>
 <tr>
  <td><a href="cloverage/sample/exercise_instrumentation.clj.html">cloverage.sample.exercise-instrumentation</a></td><td class="with-bar"><div class="covered"
-                                style="width:67.01030927835052%;
-                                        float:left;"> 195 </div><div class="not-covered"
-                                style="width:32.98969072164948%;
-                                        float:left;"> 96 </div></td>
-<td class="with-number">67.01 %</td>
+                                style="width:67.69759450171821%;
+                                        float:left;"> 197 </div><div class="not-covered"
+                                style="width:32.302405498281786%;
+                                        float:left;"> 94 </div></td>
+<td class="with-number">67.70 %</td>
 <td class="with-bar"><div class="covered"
-                                style="width:61.03896103896104%;
-                                        float:left;"> 47 </div><div class="partial"
-                                style="width:19.48051948051948%;
-                                        float:left;"> 15 </div><div class="not-covered"
+                                style="width:63.63636363636363%;
```

Comment:
I reckon all these `resources` updates deserve some review as to why they have changed.

I didn't bother checking the 'culprit' commit. Anyway all these are numeric differences are seemlingly tiny.